cmd/create.go

```
@@ -172,6 +185,77 @@ func create() {
 		log.Println("Skipping running containers and CRI-O engine already stopped.")
 	}
 
+	//
+	// Run recert pre-checks in local etcd database
+	//
+	log.Println("Running recert --dry-run to validate seed cluster can be re-certified without errors.")
+
+	// Check if the file /var/tmp/recert.done does not exist
+	if _, err = os.Stat("/var/tmp/recert.done"); os.IsNotExist(err) {
+
+		// Get etcdImageRaw available for the releaseImage variable,
+		// this is needed by recert to run an unauthenticated etcd server for dry-run pre-checks.
+		etcdImage := getEtcdImageFromStaticDefinition()
+
+		// Run unauthenticated etcd server for recert dry-run
+		// This runs a small fake unauthenticated etcd server backed by the actual etcd database,
+		// which is required before running the recert tool.
+		_, err = runInHostNamespace(
+			"podman", []string{"run",
+				"--name recert_etcd",
+				"--detach", "--rm",
+				"--network=host",
+				"--privileged",
+				"--authfile", authFile,
+				"--entrypoint", "etcd",
+				"-v", "/var/lib/etcd:/store",
+				etcdImage,
+				"--name", "editor",
+				"--data-dir", "/store"}...)
+		check(err)
```

As much as I hate Go error checking and understand your motivation for creating `check`, idiomatic Go usually doesn't hide error checking behind fatal functions like this. It's best to wrap err and return it to the caller:
```go
if err != nil {
    return fmt.Errorf("running recert etcd: %w", err)    
}
```

Then the fatal logging of the error should be done by the topmost caller - the one who has no caller to further propagate the error to. And since errors are wrapped when you use %w, you get a nice "trace" of what happened to cause the error, instead of just a vague error without context